packages/ui-kit/src/components/markdown.js

```diff
@@ -62,46 +51,57 @@ const H6 = styled.h6`
   font-weight: ${typography.fontWeights.regular};
   line-height: 1.4;
 `;
+
+/*
+The "container" styles have to be applied to containers that render markdown inside a surrounding
+visual box like a blockquote, notification box, card, or Subtitle.
+Heading margins are not set here because headings can and should not be used inside such containers.
+*/
+const containerStyles = () => css`
+  > * + * {
+    margin-top: ${dimensions.spacings.m};
+    margin-bottom: ${dimensions.spacings.m};
+  }
```

Comment:
```suggestion
  > * + * {
    margin-top: ${dimensions.spacings.m};
  }
```

This should be sufficient.

Reply:
Thanks for reminding me to the confusing parts of CSS fundamentals :-).  Vertical margin collapsing.